client/package.json

```diff
@@ -21,31 +21,33 @@
     "babel-preset-es2015": "^6.9.0",
     "babel-preset-react": "^6.11.1",
     "css-loader": "^0.23.1",
+    "d3": "^3.5.17",
     "extract-text-webpack-plugin": "^1.0.1",
     "file-loader": "^0.9.0",
     "fixed-data-table": "^0.6.1",
     "history": "^3.0.0",
     "isomorphic-fetch": "^2.2.1",
-    "keycloak-js": "1.9.8",
+    "keycloak-js": "1.9.7",
```

Comment:
yes. We're using keycloak version 1.9.7 for local development. I suppose we'll use the same version for deployments later (ping @kardan, @iperdomo).